client/scss/components/_opening_hours.scss

```diff
@@ -95,12 +125,20 @@
   display: flex;
   justify-content: space-between;
   border-bottom: 1px solid color('charcoal');
+
+  .opening-hours--light & {
+    border-bottom-color: color('pewter');
+  }
 }
 
 .opening-hours__td {
   padding: 1em 0;
   display: block;
 
+  .opening-hours--compressed & {
+    padding: $spacing-unit 0;
+  }
+
   &:last-child {
     text-align: right;
     transition: opacity 500ms ease;
```

Comment:
This seems seems like quite a lot of specific css on something that was meant to be abstracted?
Just worried about falling into image-gallery-with-100-states syndrome.